.env.vagrant

```diff
@@ -17,6 +17,7 @@ SUPPORTED_MIME_TYPES='application/pdf,image/jpeg,image/png,application/vnd.openx
 MAX_NUMBER_OF_FILES=10
 
 FILE_TRANSFER_FEATURE=1
+ILLNESS_REASON_FEATURE=1
```

Comment:
Where does the environment variable `ILLNESS_REASON_FEATURE` get read?

Reply:
The `IllnessReasonFeatureMiddleware` uses the `isFeatureEnabled` const in the `FeatureChecker` class to get the env variable

Reply:
it appends `_FEATURE` to property